ui/pages/Project/components/LoadWorkspaceDataForm.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import ReduxFormWrapper, { configuredField } from 'shared/components/form/ReduxFormWrapper'
+import { BaseBulkContent } from './edit-families-and-individuals/BulkEditForm'
+import { getEntityExportConfig } from '../selectors'
+
+const FILE_FIELD_NAME = 'uploadedFile'
+
+const mapStateToProps = (state, ownProps) => ({
+  blankExportConfig: getEntityExportConfig({ name: ownProps.workspace }, [], null, 'template', ownProps.requiredFields.concat(ownProps.optionalFields)),
```

Comment:
so if you dive into the code, `getEntityExportConfig` return a dictionary as follows:
```
 getEntityExportConfig = (project, rawData, tableName, fileName, fields) => ({
  filename: `${project.name.replace(' ', '_').toLowerCase()}_${tableName ? `${toSnakecase(tableName)}_` : ''}${fileName}`,
  rawData,
  headers: fields.map(config => config.header),
  processRow: family => fields.map((config) => {
    const val = family[config.field]
    return config.format ? config.format(val) : val
  }),
})
```
looking at this code, you see that its essentially all about how to get and process/format the project data. Since this is for a blank template, and its fine to use a basic hardcoded filename since this is empty data,  we don't need this to be a connected component that calls this selector at all. We should instead use the following constant:
```
const BLANK_EXPORT = {
  filename: 'individuals_template',
  rawData: [],
  headers: [... INDIVIDUAL_ID_EXPORT_DATA, ... INDIVIDUAL_CORE_EXPORT_DATA].map(config => config.header), // or whatever our field constants are
  processRow: val => val,
}
```

Reply:
Sounds great.